pkg/operator/ceph/object/spec.go

```diff
@@ -434,7 +435,7 @@ func (c *clusterConfig) generateService(cephObjectStore *cephv1.CephObjectStore)
 		ObjectMeta: metav1.ObjectMeta{
 			Name:      instanceName(cephObjectStore.Name),
 			Namespace: cephObjectStore.Namespace,
-			Labels:    getLabels(cephObjectStore.Name, cephObjectStore.Namespace, true),
+			Labels:    getLabels(cephObjectStore.Name, cephObjectStore.Name, cephObjectStore.Namespace, true),
```

Comment:
@BlaineEXE I am not sure what to pass here for the `generateService`, so passed its name, in place of daemonID.